### ISSUE TITLE
Premature Termination After Sanity Check Failures

### ISSUE THREAD

**Issue by DarioS** [PapenfussLab/gridss#311]
I have seen log files with failures before, but the processing continued to the end and a VCF file was produced. But, a pair of samples I tested recently seems to have caused GRIDSS to terminate and no VCF was produced. The log file is [attached](https://github.com/PapenfussLab/gridss/files/4352282/gridss.full.20200318_161752.gadi-cpu-clx-0376.gadi.nci.org.au.54781.log).

```
   NCPUs Requested:    8                      NCPUs Used: 8               
                                           CPU Time Used: 35:37:37                                   
   Memory Requested:   190.0GB               Memory Used: 180.84GB        
   Walltime requested: 25:00:00            Walltime Used: 06:42:21        
   JobFS requested:    100.0MB                JobFS used: 0B       
```

**Comment by d-cameron** [comment]
The streaming window size fix included in the latest release exposed another bug that was mostly hidden. When I updated GRIDSS to include the RP anchoring reads in the assembly, I didn't update the evidence tracking to handling multiples reads with the same identifier. This has been fixed in dev (abc481e422ad25b118c1e61259f481752b2aeb81), but not yet released as I'm tracking down one final assembly issue that my regression testing picked up.

A fix for this will be in the next release (ETA is next week).
